cmd/di.go

```diff
@@ -194,6 +194,8 @@ func (di *Dependencies) Bootstrap(nodeOptions node.Options) error {
 		return err
 	}
 
+	netutil.ClearStaleRoutes()
```

Comment:
After we merge https://github.com/mysteriumnetwork/node/pull/2302 if would be better to move this to suppervisor command to clear routes.